catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/ops/PaimonTableOps.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon.ops;
+
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.utils.CatalogUtils.loadCatalogBackend;
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.utils.TableOpsUtils.buildSchemaChanges;
+
+import com.datastrato.gravitino.catalog.lakehouse.paimon.PaimonConfig;
+import com.datastrato.gravitino.rel.TableChange;
+import java.util.List;
+import java.util.Map;
+import org.apache.paimon.catalog.Catalog;
+import org.apache.paimon.catalog.Catalog.ColumnAlreadyExistException;
+import org.apache.paimon.catalog.Catalog.ColumnNotExistException;
+import org.apache.paimon.catalog.Catalog.DatabaseAlreadyExistException;
+import org.apache.paimon.catalog.Catalog.DatabaseNotEmptyException;
+import org.apache.paimon.catalog.Catalog.DatabaseNotExistException;
+import org.apache.paimon.catalog.Catalog.TableAlreadyExistException;
+import org.apache.paimon.catalog.Catalog.TableNotExistException;
+import org.apache.paimon.catalog.Identifier;
+import org.apache.paimon.schema.Schema;
+import org.apache.paimon.table.Table;
+import org.apache.paimon.utils.Pair;
+
+/** Table operation proxy that handles table operations of an underlying Paimon catalog. */
+public class PaimonTableOps implements AutoCloseable {
```

Comment:
`IcebergTableOps` is used to support GravitinoIcebergCatalog  and Iceberg REST server. Is it necessary to add `PaimonTableOps` for `Paimon`?